src/App.vue

```diff
@@ -45,6 +47,12 @@ onMounted(() => {
   const ms15Minutes = 15 * 60 * 60 * 1000;
   buildDateIntervalId = setInterval(updateRelativeOsimBuildDate, ms15Minutes);
   updateRelativeOsimBuildDate();
+
+  addToast({
```

Comment:
I'm worried that showing this notification each time the app is loaded might be too cumbersome to the users, maybe displaying it on login is enough, **but** if is required for PIA, then its ok